opensync-ext-cloud/src/main/java/com/telecominfraproject/wlan/opensync/external/integration/utils/MqttStatsPublisher.java

```diff
@@ -1781,8 +1784,8 @@ ChannelInfo createChannelInfo(long equipmentId, RadioType radioType, List<Survey
         channelInfo.setTotalUtilization(totalUtilization.intValue());
         channelInfo.setWifiUtilization(totalUtilization.intValue() - totalNonWifi.intValue());
         channelInfo.setBandwidth(channelBandwidth);
-        if (surveySampleList.size() > 0) {
-            channelInfo.setNoiseFloor((int) Math.round(DecibelUtils.getAverageDecibel(noiseArray)));
+        if (noiseList.size() > 0) {
+            channelInfo.setNoiseFloor((int) Math.round(DecibelUtils.getAverageDecibel(toIntArray(noiseList))));
```

Comment:
Use getAverageDecibel(Collection<Integer> decibels) rather than convert to int[]?